lib/OpenQA/WebAPI.pm

```diff
@@ -153,6 +153,15 @@ sub startup {
     $auth_module->import('auth_config');
     auth_config($self->config);
 
+    # Check and create if needed system user
+    $schema->resultset('Users')->find_or_create(
+        {
+            username => 'system',
+            email    => 'noemail@open.qa',
+            fullname => 'openQA system user',
+            nickname => 'system'
+        });
+
```

Comment:
How do I do this? Upgrade part I get, but deploy. Add a deployment hook in Users schema? Or `deploy/system_user.pl` under _common?